LWCOSC/force-app/main/default/lwc/doPagination/doPagination.js-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>50.0</apiVersion>
+  <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__RecordPage</target>
```

Comment:
We should be very careful of what we are exposing the component to

Reply:
Point Noted, As discussion No action require